app/src/routes/v1/stats.js

```diff
@@ -0,0 +1,29 @@
+const statsRouter = require('express').Router();
+const Problem = require('api-problem');
+//const log = require('npmlog');
+
+const stats = require('../../components/stats');
+
+// fetches all the service clients for all acronyms in all KC realms
+statsRouter.get('/serviceClients', async (req, res) => {
+
+  // requires user role "GETOK_ADMIN"
```

Comment:
i see the bearer token in the header of the request (req) but i still havent found a good way to check the current users keycloak role :(

Reply:
It could look something like this:
`statsRouter.get('/serviceClients', keycloak.protect('realm:GETOK_ADMIN'), async (req, res) => {`
Examples from their docs: https://github.com/keycloak/keycloak-quickstarts/blob/latest/service-nodejs/app.js